chatbot-core/backend/.dockerignore

```diff
@@ -0,0 +1,5 @@
+.vscode/
+.python-version
+.gitignore
+README.md
+requirements.txt
```

Comment:
If using `uv`, consider removing `requirements.txt`. Same for building Docker image (copy `uv.lock` into Docker container and run `uv.sync`).